src/components/auth/SignUp.js

```diff
@@ -66,64 +67,77 @@ const SignUp = (props) => {
+                                />
+                            </div>
+                        {/* </div> */}
+
+                        <div className="py-2 pb-2">
+                            <label htmlFor="email" className="text-sm block text-xl">Email</label>
+                            <input
+                                className="textshadow appearance-none border rounded w-full py-4 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-gray-500"
                                 type="text"
                                 name="email"
                                 value={credentials.email}
                                 onChange={handleChange}
                             />
                         </div>
-                        <div className="px-4 pb-4">
+
+                        <div className="py-2 pb-2">
                             <label 
                                 htmlFor="password" 
-                                className="text-sm block font-bold pb-2">
-                                PASSWORD
+                                className="text-sm block text-xl">
+                                Password
                             </label>
                             <input
-                                className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-blue-300"
-                                placeholder="Enter your password"
+                                className="textshadow appearance-none border rounded w-full py-4 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-gray-500"
                                 type="password"
                                 name="password"
                                 value={credentials.password}
                                 onChange={handleChange}
                             />
                         </div>  
+                        
                         <button 
-                            className="bg-indigo-900 hover:bg-green-400 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline" 
+                            className="hover:bg-blue-300 bg-gray-500 py-2 px-4 my-4 w-full h-16 rounded text-white text-3xl focus:outline-none focus:shadow-outline" 
                             type="submit" 
                             onClick={signUp}>
                             Sign Up
                         </button>
-                        <Link to="/login" className="border-t block no-underline hover:underline py-2 text-grey-darkest hover:text-black md:border-none md:p-0">
-                            Already a Member? Login here
-                        </Link>
+                        
+                        <div>Already a Member?<Link to="/login" className="text-blue-700 hover:font-bold"> Log In</Link></div>
+                        <br/><br/><br/>
```

Comment:
Try to avoid `<br/>` for mimicking spacing/padding if you can. All that should be handled in CSS... or in this case: `pb-3` or some such.